src/infosquito/es_crawler.clj

```diff
@@ -20,7 +20,7 @@
   (let [res (esd/search es index (name item-type)
               :query       (q/match-all)
               :_source     ["_id"]
-              :sort        ["_doc"]
+              :sort        ["id"]
```

Comment:
This is so we can predict what's sensible to prefetch. It doesn't seem to slow down the search particularly (or at least, if it does, it's dwarfed by the speedups due to the cache)